.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+### Type of change
+
+- [ ] bug fix
+- [ ] new feature
+- [ ] documentation
+- [ ] other (describe)
+
+### Scope of changes
+
+Briefly describe the scope of changes. Include links to ShortCut or Jira ticket if applicable.
+
+### Acceptance criteria
+
+Describe how reviewers can test this change to be sure that it works correctly. Add a checklist if possible.
+
+### Author checklist
+
+- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
+- [ ] I have added migrations for any changes to the database
+- [ ] My migrations can be run before the code is deployed without causing any issues
```

Comment:
Not sure if these two make sense for our project since we don't have a relational database backend. Perhaps we could replace these with something more like:
- [ ] I have updated the dependencies list
- [ ] I have recompiled and included new protocol buffers to reflect changes I made
- [ ] I have added new test fixtures as needed to support added tests

etc